py3status/modules/file_status.py

```diff
@@ -56,10 +57,6 @@ class Meta:
             ],
         }
 
-    def post_config_hook(self):
-        if self.path:
-            self.path = expanduser(self.path)
-
```

Comment:
Keep this so we can do `expanduser(path)` once instead of every refresh interval.

Reply:
I have a doubt about this, refresh is for example when I `py3-cmd resfresh file_status`? I thought this part was only excecuted at the py3status start/restart.

Reply:
Not understand this sorry, could you rephrase? "I'd clear this with the fam first."

Reply:
Have a discussion with **tobes** and/or **ultrabug**. They our fam.

Reply:
Oh sorry, fam = familly >_>